tests/core/stackAlloc.cpp

```diff
@@ -2,20 +2,21 @@
 
 int main() {
   EM_ASM({
-    var size = 100;
+    var size = 128;
     var before;
     before = stackSave();
     var x = stackAlloc(size);
     var y = stackAlloc(size);
     var direction = y > x ? 1 : -1;
+    assert(x % 16 == 0, "allocation must have 16-byte alignment");
     assert(x == Math.min(before, before + direction*size), "allocation must return the start of the range allocated");
     var z = stackAlloc(size);
     assert(x != y && y != z && x != z, "allocations must be unique");
     assert((y - x)*(z - y) > 0, "allocations must be in the same direction");
     // no overlaps
-    function notInRange(value, oneSide, otherSide) {
-      assert(!(Math.min(oneSide, otherSide) <= value &&
-               value <= Math.max(oneSide, otherSide)), value + " must not be in the range " + [oneSide, otherSide]);
+    function notInRange(value, begin, end) {
+      if (begin < end) assert(!(value >= begin && value < end), value + " must not be in the range [" + [begin, end] + ")");
```

Comment:
Wouldn't this get printed as `must not be in the range [[123, 456])`?

Reply:
er, yeah, fixed.